src/lib/services/events-service.ts

```diff
@@ -52,11 +52,118 @@ export const fetchRawEvents = async ({
   return response.history.events;
 };
 
-export const fetchEvents = async (
+export const fetchPartialRawEvents = async ({
+  namespace,
+  workflowId,
+  runId,
+  sort,
+}: FetchEventsParameters): Promise<HistoryEvent[]> => {
+  const route = await routeForApi(`events.${sort}`, {
+    namespace,
+    workflowId,
+    runId,
+  });
+
+  const response = await requestFromAPI<GetWorkflowExecutionHistoryResponse>(
+    route,
+    {
+      request: fetch,
+      params: { maximumPageSize: '20' },
```

Comment:
Is 20 a good number here? In reality it will return 40ish events (the maximumPageSize is the batch size, the number of events is unknown). We could probably make this 10 or lower